### PR TITLE
Feature/applics 81 register to have your say translation 3 of 3

### PR DESCRIPTION
**Pull request by edpackard-work**
## Describe your changes

https://pins-ds.atlassian.net.mcas.ms/jira/software/c/projects/APPLICS/boards/242?selectedIssue=APPLICS-81

Update views, routers and validators for agent journey on register to have your say journey.
Also some translation fixes for other parts of the register to have your say journey.

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/src/pages/projects/register/agent/router.js [hunk 1]
```diff
@@ -181,11 +178,15 @@ registerAgentRouter.post(
 
 registerAgentRouter.get(
 	registerAgentOrgNameURL,
+	addCommonTranslationsMiddleware,
```
Comment (cprieststephens) [inline packages/forms-web-app/src/pages/projects/register/agent/router.js:181]: Could the middleware that's being used on multiple routes be added at router level?

`registerAgentRouter.use(addCommonTranslationsMiddleware, addRegisterTranslationsMiddleware)`

**Review by cprieststephens — CHANGES_REQUESTED** [review]

File: packages/forms-web-app/src/pages/projects/register/agent/router.js (hunk 1 above)
Comment (edpackard-work) [reply packages/forms-web-app/src/pages/projects/register/agent/router.js]: good call - have tidied this up across the register to have your say routes

**Review by cprieststephens — APPROVED** [review]

**Pull request merged**